test/Chainweb/Test/Pact/ModuleCacheOnRestart.hs

```diff
@@ -76,29 +76,28 @@ tests rdb =
       withMVarResource mempty $ \iom ->
       withEmptyMVarResource $ \rewindDataM ->
       withTestBlockDbTest testVer rdb $ \bdbio ->
-      withTemporaryDir $ \dir ->
+      withTempSQLiteResource $ \ioSqlEnv ->
       testGroup label
-      [
-        withPact' bdbio dir iom testInitial (testCase "testInitial")
+      [ testCase "testInitial" $ withPact' bdbio ioSqlEnv iom testInitial
```

Comment:
The type of `withPact'` is simplified, which allows a more straight forward invocation of the tests.